src/main/java/com/minecolonies/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -5,6 +5,7 @@
 import com.minecolonies.colony.ColonyView;
```

Comment:
There are 65 changes to this class, and they are all formatting changes. Please don't run class level formatters.